3rdparty/irodsphp/prods/src/ProdsStreamer.class.php

```diff
@@ -31,6 +31,15 @@ class ProdsStreamer
 	 * @access private
 	 */
 	private $file;
+	
+	/**
+         * Metadata of the file/directory
+         *
+         * @access public
+         */
+        public $metadata;
+
+        const DEFAULT_RESCNAME = "df.bluefern.canterbury.ac.nz";
```

Comment:
This could be generalized too somehow by exposing it in a config file instead of the base class?...